### DISCUSSION TITLE
Best Practices for Apollo Federation -- Mutations

### DISCUSSION THREAD

**Discussion by rishabhp58** [apollographql/apollo-server#4163] [General]
I'm currently working with two federated schemas. Is it possible for me to use mutations to change data against both of the data sources? Using a single mutation?

Is it best practice/recommendation that federation be used for only Query types and not for mutations? i.e. for mutations make a separate/independent calls to service x and service y.

Any guidance on this is appreciated. Thanks!

**Comment by iamfotx** [comment]
Is there any update? @theshabh58 how did you end up solving the issue?
